app/views/spree/orders/_summary.html.haml

```diff
@@ -1,3 +1,5 @@
+= display_footer = true if display_footer.nil?
```

Comment:
Doesn't this print `true` when display_footer is nil? I think that it should be this:
```suggestion
- display_footer = true if display_footer.nil?
```

Reply:
Oh yes, good catch!!!